### PR TITLE
[FIX] saas_portal: fix link for credential url

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: saas_portal/data/mail_template_data.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
             <field name="body_html"><![CDATA[
 
             <p>Web:
-            <a href="http://${object.name}">http://${object.host}</a>
+            <a href="http://${object.host}">http://${object.host}</a>
```
Comment (yelizariev) [inline saas_portal/data/mail_template_data.xml]: add support for https

**Comment by iledarn** [comment]
@yelizariev , done

File: saas_portal/models/saas_portal.py [hunk 2]
```diff
@@ -548,6 +549,15 @@ def _compute_active(self):
         for record in self:
             record.active = record.state != 'deleted'
 
+    @api.multi
+    @api.depends('server_id.request_port', 'server_id.request_scheme', 'host')
+    def _compute_public_url(self):
+        for record in self:
+            public_url = "%s://%s" % (record.server_id.request_scheme, record.host)
+            if record.server_id.request_port and record.server_id.request_port != 80:
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: For https port is 443

File: saas_portal/models/saas_portal.py [hunk 3]
```diff
@@ -548,6 +549,15 @@ def _compute_active(self):
...
+                public_url = public_url + ':' + str(record.server_id.request_port)
+            record.public_url = public_url
```
Comment (yelizariev) [inline saas_portal/models/saas_portal.py]: probably, trailing slash is needed

```
        record.public_url = public_url + '/'
```

File: saas_portal/models/saas_portal.py (hunk 2 above)
Comment (yelizariev) [reply saas_portal/models/saas_portal.py]:
```
         if scheme == 'http' and port_str == '80' or scheme == 'https' and port_str == '443':
```

**Comment by yelizariev** [comment]
@iledarn Also, you can `update _create_new_database` and use `public_url` there

```
        'r': '%s://%s%s/web' % (scheme, client.host, port_str),
```

**Comment by iledarn** [comment]
@yelizariev , done

**Pull request merged**